common/models/mongo-queryable.js

```diff
@@ -127,7 +127,7 @@ module.exports = function (MongoQueryableModel) {
     // mode: additional query expression
 
     let allMatch = [];
-    Object.keys(fields).map(function (key) {
+    Object.keys(fields).forEach(function (key) {      
```

Comment:
Extra white space at the end of line 130 should be removed

Reply:
Done.